website/docs/reference/global-configs/warnings.md

```diff
@@ -2,129 +2,206 @@
 title: "Warnings"
 id: "warnings"
 sidebar: "Warnings"
+toc_max_heading_level: 2
+intro_text: "dbt's warnings can be converted to errors using --warn-error flag or WARN_ERROR config, with granular control through options."
```

Comment:
you can use either_`--warn-error` or `--warn-error-options` - using `--warn-error` is the same as doing `--warn-error-options '{"error": "all"}'`, it promotes _all_ warnings to errors